test/jsonapi/view_test.exs

```diff
@@ -43,6 +43,18 @@ defmodule JSONAPI.ViewTest do
     use JSONAPI.View, type: "cars", namespace: ""
   end
 
+  defmodule DynamicView do
+    use JSONAPI.View
+
+    def type, do: "dyns"
+
+    def fields, do: [:static_fun, :static_field, :dynamic_1, :dynamic_2]
+
+    def static_fun, do: "static_fun!"
```

Comment:
This is supposed to be the 2-arity function that gets called when defined with `get_field/3` as a fallback right?

Reply:
Ah, yes 🤦 Fixed.